wire/msgtx.go

```diff
@@ -83,6 +83,21 @@ const (
 	// of transaction signatures + LockTime 4 bytes + Expiry 4 bytes + min
 	// input payload + min output payload.
 	minTxPayload = 4 + 1 + 1 + 1 + 4 + 4
+
+	// freeListMaxScriptSize is the size of each buffer in the free list
+	// that	is used for deserializing scripts from the wire before they are
+	// concatenated into a single contiguous buffers.  This value was chosen
+	// because it is slightly more than twice the size of the vast majority
+	// of all "standard" scripts.  Larger scripts are still deserialized
+	// properly as the free list will simply be bypassed for them.
+	freeListMaxScriptSize = 512
```

Comment:
Might want to consider tweaking this number since I don't believe standard Decred transaction script is the same as Bitcoin.